requirements_develop.txt

```diff
@@ -1,5 +1,5 @@
 autoflake>=2.3.0
```

Comment:
![image](https://github.com/RimSort/RimSort/assets/2488703/a28bfb1b-ce80-41c4-9119-23735d602019)
it also replaces autoflake